### ISSUE TITLE
How practical is web components into building real world sites

### ISSUE THREAD

**Issue by AmitKumarDas** [skatejs/skatejs#83]
This is about using web components in production. Is web component the silver bullet ?
Or it needs other libraries & stuff to make things work.

I am referring to a recent article about web components vs React like libraries.
http://jlongster.com/Removing-User-Interface-Complexity,-or-Why-React-is-Awesome

The latter try to understand the issues we all have building a web site and hence the library(s).

In other words, can skatejs complement or compete etc with libraries like Mercury (https://github.com/Raynos/mercury), Mithril (http://lhorie.github.io/mithril/), etc. After all these libraries try to solve the entire gamut of problems that web developers face.

**Comment by treshugart** [comment]
Hey @AmitKumarDas, just following up on this since it's been awhile. Skate is going 1.0 and now has an opinion on rendering using a virtual DOM provided by [Incremental DOM](https://github.com/google/incremental-dom).
